operation/fhir-operation-versions/README.md

```diff
@@ -0,0 +1,119 @@
+# $versions operation
+
+The [$versions](https://hl7.org/fhir/capabilitystatement-operation-versions.html) operation allows a client to find out what versions a server supports.
+
+A client invokes the operation with no parameters and the server returns the list of supported versions, along with the default version it will use if no [fhirVersion parameter](https://hl7.org/fhir/http.html#version-parameter) is present. The response format is requested via `Accept` HTTP header, which may be set to `application/fhir+json`, `application/fhir+xml`, `application/json`, `application/xml`. If the `Accept` HTTP header is not set, the default response format is `application/fhir+json`.
+
+## Examples
+
+### JSON (FHIR)
+
+Request:
+
+```
+    GET [base]/$versions
+    Accept: application/fhir+json
+    [other headers]
+```
+
+Response:
+
+```json
+HTTP/1.1 200 OK
+[other headers]
+
+{
+    "resourceType": "Parameters",
+    "parameter": [{
+    "name": "version",
+    "valueString": "4.0"
```

Comment:
maybe indent with 4 spaces so the structure stands out better and matches our actually pretty-printed response

Reply:
Fixed.